Add random tribe background images from CDN data

Jagjeet opened the pull request:
#### Proposed Changes
* Add random background image ids to the mock data  (using the tribe dump file (10-20-2018) provided by Mikael)
* This lets the mock tribes data use random images currently used by production tribes from the CDN

#### Testing Instructions

* Add mock tribe data via `npm run test-data:tribes 50`
* Add some users via `npm run test-data:users 10 admin1`
* Login as admin1 (L: admin1 PW: password123)
* Verify tribes have background images

Fixes 
This was one more issue related to #462.

nicksellen commented:
Great! Seed data for dev environment makes me very happy.

Any resistance to naming it _seed_ data as opposed to _test_ data? (_test_ data sounds like it's used for the tests) - see https://en.wikipedia.org/wiki/Database_seeding

Jagjeet commented:
Ha! I just had this thought to as I am also working on a test PR! I'll pull together a PR for that too.

simison commented on bin/fillTestData/Tribes.js:
```diff
@@ -36,7 +42,7 @@ var addTribes = function (max) {
         tribe.modified = Date.now();
         tribe.public = true;
-        // tribe.image_UUID = faker.random.uuid();
+        tribe.image_UUID = tribesData[random(tribesData.length)].image_UUID;
```
What do you think if we only include the UUIDs as an array here, since rest of the data isn't used anyway?

```
var UUIDs = [
  "171433b0-853b-4d19-a8b4-44def956696d",
  "e69eb05f-773f-423c-9246-43629b5a8baf",
  ...
];

simison commented on bin/fillTestData/Tribes.js:
```diff
@@ -4,8 +4,14 @@ var path = require('path'),
 
+var random = function (max) {
+  return Math.floor(Math.random() * max);
+};
```
Since we have `lodash` at our hands,  `_.sample` could work smoothly here?

https://lodash.com/docs/4.17.10#sample

Jagjeet commented:
Thanks for the suggestions. I went ahead with the changes. There wasn't really a good use case for keeping the json data so went ahead and removed it and switched to an array of UUIDs.
